uefi-services/README.md

```diff
@@ -1,4 +1,4 @@
 # uefi-services
 
 WARNING: `uefi-services` is deprecated. Functionality was moved to
-`uefi::helpers::init` in `uefi` ´v0.28.0`.
+`uefi::helpers::init` in `uefi@v0.28.0`.
```

Comment:
nit: Unrelated to the rest of the PR. Not a big deal, but for me it helps review if the PR is as focused/small as possible.

Reply:
It is in a dedicated commit. Is this ok?

Reply:
Yep it's fine. Mostly I just don't think github's UI for commit-by-commit reviews is very good (this is one area where I prefer [gerrit](https://www.gerritcodereview.com/)), so in general I would prefer things like this in a separate PR, even if it's just a one-line doc change. However, not a big deal in this case.